src/storage/mvcc/txn.rs

```diff
@@ -857,7 +863,7 @@ mod tests {
             pk,
             &Options::default(),
         ).unwrap();
-        engine.write(&ctx, txn.into_modifies()).unwrap();
+        write_unless_empty_modify(engine, &ctx, txn.into_modifies());
```

Comment:
Can `txn.into_modifies()` be empty? If it's empty, isn't it a bug?

Reply:
Yes, it can.

Reply:
Then why not fix it?